packages/workflow-compile/index.js

```diff
@@ -33,6 +33,12 @@ async function compile(config) {
       const compileMethod =
         config.all === true || config.compileAll === true
           ? Compile.all
+          : config.specificFiles
+          ? config =>
+              Compile.sourcesWithDependencies({
+                paths: config.specificFiles,
+                options: config
+              })
```

Comment:
I may have broken this. Need to test that `truffle compile <vyper-contract>` doesn't cause compile-solidity to explode.